lib/scenic/color.ex

```diff
@@ -535,7 +535,7 @@ defmodule Scenic.Color do
 
     s =
       case delta do
-        0 -> 0
+        0.0 -> 0.0
         d -> d / (1 - abs(2 * l - 1))
       end
```

Comment:
Maybe we want to replace the `case` here with a `cond` instead that tests against some small epsilon value instead of directly matching for `0.0`?

Reply:
Yeah I'm fine with that approach, and given how floats work it is probably more reliable!

Reply:
Okay, here's an implementation that uses epsilon: b22c070719cdc963bc17e497968142e71a2ab51f

However I wasn't able to find any recommendations for an epsilon value for Elixir/Erlang so I just used the value from JS: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/EPSILON

Do you have a recommendation for an epsilon value to use?

Reply:
I think in my own libraries I've used `1e-6` or something like that, but I'm not super picky.